src/routes/index.tsx

```diff
@@ -2,36 +2,32 @@ import React from "react";
 import { Redirect, Route, Switch } from "react-router-dom";
 
 import NotFoundPage from "~/pages/error/NotFound";
-import BoardingFarmer from "~/pages/public/BoardingFarmer";
-import ForgotPassword from "~/pages/public/ForgotPassword";
-import Home from "~/pages/public/Home";
-import Login from "~/pages/public/Login";
-import Map from "~/pages/public/Map";
+import { BoardingFarmer, ForgotPassword, Home, Login } from "~/pages/public";
 import SplashScreen from "~/pages/public/SplashScreen";
+import NavigationWrapper from "~/routes/NavigationWrapper";
 
 export default function AppRoutes() {
   return (
     <Switch>
       {/*Public Routes*/}
       <Route exact path="/" component={Home} />
-      <Route exact path="/login" component={Login} />
+      <Route exact path="/signin" component={Login} />
 
-      <Route exact path="/map" component={Map} />
       <Route exact path="/boarding-farmer" component={BoardingFarmer} />
       <Route exact path="/forgot-password" component={ForgotPassword} />
 
-      <Route exact path="/404" component={NotFoundPage} />
-
       {/* Screens - only for presentational purposes */}
       <Route exact path="/screens/splash" component={SplashScreen} />
+      {/*
+       <Route path="/profile" component={userIsAuthenticated(Profile)}/>
+       */}
+
+      <NavigationWrapper />
+
+      <Route exact path="/404" component={NotFoundPage} />
 
       {/*Redirect*/}
-      <Redirect
-        push
-        to={{
-          pathname: "/404"
-        }}
-      />
+      <Redirect push to={{ pathname: "/404" }} />
```

Comment:
**Z30:** Das object in der `to` prop könnte man außerhalb in einer const definieren, um zu verhindern, dass es bei jedem Rerender neu erstellt wird. Nur eine kleine Optimierung.